test/cli-utils-browser.spec.js

```diff
@@ -1,7 +1,7 @@
 /*jshint jasmine: true, node: true */
 'use strict';
 
-const url = require('url');
+const urlParser = require('url');
```

Comment:
Renamed this import because eslint was throwing the "shadowed variable `url`" in the code below.